### PR TITLE
MET-1769 Added sampling checkbox

### PR DESCRIPTION
**Pull request by andyjmaclean**

### CODE REVIEW

File: src/app/dataset/workflow/workflow.component.html [hunk 1]
```diff
@@ -430,6 +429,60 @@
...
         /></label>
+        <div
+          class="form-group"
+          *ngIf="
+            !workflowForm.get('pluginLINK_CHECKING')!.disabled &&
+            workflowForm.get('pluginLINK_CHECKING')!.value
+          "
+        >
+          <div class="form-group-radios">
+            <div>
```
Comment (mirjamverloop) [inline src/app/dataset/workflow/workflow.component.html]: Is this div necessary?

File: src/app/dataset/workflow/workflow.component.html [hunk 2]
```diff
@@ -430,6 +429,60 @@
...
+                  type="radio"
+                  formControlName="performSampling"
+                  value="true"
+                  [checked]="performSampling === true"
+                  (change)="changeLinkCheckSampling(true)"
+                />
+                {{ 'checksample' | translate }}
+              </label>
+            </div>
+            <div>
```
Comment (mirjamverloop) [inline src/app/dataset/workflow/workflow.component.html]: Is this div necessary?

File: src/app/dataset/workflow/workflow.component.html (hunk 1 above)
Comment (andyjmaclean) [reply src/app/dataset/workflow/workflow.component.html]: This was copied and pasted from the harvest url radios.  I've removed the redundant div elements from both places.

File: src/app/dataset/workflow/workflow.component.html (hunk 2 above)
Comment (andyjmaclean) [reply src/app/dataset/workflow/workflow.component.html]: This was copied and pasted from the harvest url radios.  I've removed the redundant div elements from both places.

**Pull request merged**